internal/service/ec2/ec2_fleet.go

```diff
@@ -638,6 +644,7 @@ func resourceFleetRead(ctx context.Context, d *schema.ResourceData, meta interfa
 	ignoreTagsConfig := meta.(*conns.AWSClient).IgnoreTagsConfig
 
 	fleet, err := FindFleetByID(ctx, conn, d.Id())
+	excessCapacityTerminationPolicy := d.Get("excess_capacity_termination_policy")
```

Comment:
Using this for backwards compatibility to track if the excess_capacity_termimation_policy was initially sent. This would help prevent breaking change due to the removal of the Default.